core/lib/Thelia/TaxEngine/TaxEngine.php

```diff
@@ -128,8 +128,10 @@ public function getDeliveryCountry()
                     $this->taxState = $currentDeliveryAddress->getState();
                 } else {
                     $customerDefaultAddress = $customer->getDefaultAddress();
-                    $this->taxCountry = $customerDefaultAddress->getCountry();
-                    $this->taxState = $customerDefaultAddress->getState();
+                    if (isset($customerDefaultAddress)) {
+											$this->taxCountry = $customerDefaultAddress->getCountry();
```

Comment:
Could you please use spaces instead of tabs for indentation ?

Reply:
Yes of course. I've replaced tabs with spaces